Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN mix test --no-compile
 
 # Build the escript
 RUN mix escript.build
-RUN mv exercism_formatter /opt/test-runner/bin
+RUN mv exercism_test_helper /opt/test-runner/bin
```

Comment:
I don't know anything about Erlang, but are there different Docker images? As in, one that has just the runtime and one that also has the SDK to build? Ignore if this does not make any sense.

You could consider using a [Docker multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) to only include the files needed to run the rest runner.

Reply:
escripts still require an erlang runtime on the system, so having multistage build doesn't win much..

Reply:
It's a good suggestion though in general to consider

Reply:
Ah okay, just wanted to verify. Thanks for explaining!